File: Feitos em C/2963.c
#include <stdio.h>
int main(){
    int ent,x;
    scanf("%d",&ent);
	int vet[ent];
	for(x=0;x<ent;x++){
	    scanf("%d",&vet[x]);
	}
	int igual=0,maior=0;
	for(x=1;x<ent;x++){
		if(vet[0]==vet[x]){
			igual++;
		}
	}
	if(igual==ent+1){
	    printf("S\n");
	}
	else{
		for(x=1;x<ent;x++){
			if(vet[0]<vet[x]){
				maior++;
			}
		}
		if(maior!=0){
			printf("N\n");
		}
		else{
			printf("S\n");
		}
	}
    return 0;
}

File: Feitos em Java/2963.java
import java.io.IOException;
import java.util.Scanner; 
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner entrada = new Scanner (System.in);
        int ent=entrada.nextInt();
    	int[] vet;
            vet = new int[ent];
    	for(int x=0;x<ent;x++){
            vet[x]=entrada.nextInt();
    	}
    	int igual=0,maior=0;
    	for(int x=1;x<ent;x++){
            if(vet[0]==vet[x]){
                igual++;
            }
    	}
    	if(igual==ent+1){
            System.out.println("S");
    	}
    	else{
            for(int x=1;x<ent;x++){
                if(vet[0]<vet[x]){
                        maior++;
                }
            }
            if(maior!=0){
                System.out.println("N");
            }
            else{
                System.out.println("S");
            }
    	}
    }
}